### PR TITLE
Add replaceSlugs filter for ComposerMakeCommand

### PR DESCRIPTION
**Pull request by Log1x**

### CODE REVIEW

File: src/Acorn/Console/Commands/ComposerMakeCommand.php [hunk 1]
```diff
@@ -65,19 +67,36 @@ protected function buildClass($name)
...
      * @param  string  $stub
      * @param  array   $views
-     * @return string
+     * @return $this
      */
     protected function replaceViews($stub, $views)
     {
         $views = implode("',\n        '", $views);
-        return str_replace('DummyViews', empty($views) ? '//' : "'{$views}'", $stub);
+        $stub = str_replace('DummyViews', empty($views) ? '//' : "'{$views}'", $stub);
```
Comment (QWp6t) [inline src/Acorn/Console/Commands/ComposerMakeCommand.php]: so what's actually being done with `$stub`?

File: src/Acorn/Console/Commands/ComposerMakeCommand.php [hunk 2]
```diff
@@ -65,19 +67,36 @@ protected function buildClass($name)
...
+    /**
+     * Replace the slugs for the given stub.
+     *
+     * @param  string  $stub
+     * @param  string  $name
+     * @return $this
+     */
+    protected function replaceSlugs($stub, $name)
+    {
+        $stub = str_replace('DummySlug', Str::slug($name), $stub);
```
Comment (QWp6t) [inline src/Acorn/Console/Commands/ComposerMakeCommand.php]: What's being done with `$stub`? And where is `DummySlug` set?

**Comment by Log1x** [comment]
i sleepy i come fix & explain tmrw

**Comment by Log1x** [comment]
ok I changed them back to returning strings. not sure if we wanted to make a separate method for running our "filters" (replacing views, slugs) – as far as where `DummySlug` is being used atm– in Composer but likely will replace a few use-cases in Acorn shortly too.

**Comment by QWp6t** [comment]
What I mean is, there isn't a `DummySlug` anywhere in [composer.stub](https://github.com/roots/acorn/blob/master/src/Acorn/Console/Commands/stubs/composer.stub), so it seems as though that method isn't actually doing anything at the moment, right?

**Comment by Log1x** [comment]
Actually, I'm an idiot and confused myself. We should just move these to [GeneratorCommand.php](https://github.com/roots/acorn/blob/master/src/Acorn/Console/Commands/GeneratorCommand.php)... I'm closing this and will redo.

**Pull request closed without merging**